CheoR/levelup
# Cr expose event endpoints

# Description

Add event view set and serializsers to expose event endpoints.

## Type of change


# Testing Instructions

`git fetch --all`
`git checkout cr-expose-event-endpoints`
`python3 manage.py runserver`

In Postman, confirm following endpoints

`http://8000/events`
`http://8000/events/1`

For client-side instructions, view client-side pull request.

# Checklist:
